RDF-Pipeline/t/tests/0058_Test-tools-diagram-sparql-lists/setup-files/test-in.txt

```diff
@@ -0,0 +1,20 @@
+# Test data for diagram-sparql
+
+:enrich-merge a p:GraphNode ;
+  p:inputs ( :enrich-address :enrich-name :enrich-sig :enrich-icd9tosnomed ) .
+
+:s1 :p ().
```

Comment:
Perhaps a comment about this testing blank nodes might be good here - helpful for the non-RDF gurus.